### ISSUE TITLE
Managing Users in Roles

### ISSUE THREAD

**Issue by jwhitener6** [aspnet/AspNetCore.Docs#11841]
This tutorial and the subsequent Authorization articles seem to cover well the creation of roles, policy-based authorization and even resource-based ones, but I don't see anything about adding users to a specific role. In fact the "Add or Remove a user to a role" is blank, citing only a small forum issue about muting privileges.

I understand its dependent on the back end authentication method. But I can't find any information on managing the role membership.

In my particular scenario I am trying to use windows authenticated logins, and place users into roles managed within the application (not through AD). Though I can't see how to even manage that in the lab provided, other than editing the SeedDB initializer, which is impractical in production. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b66f8ffa-699c-8b11-0cab-805f5953805f
* Version Independent ID: 7c151ed5-fd78-84fd-a08a-88912990c4ca
* Content: [Create an ASP.NET Core app with user data protected by authorization](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/secure-data?view=aspnetcore-2.2)
* Content Source: [aspnetcore/security/authorization/secure-data.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/security/authorization/secure-data.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Lokesh-Eisen** [comment]
kind of late but from this document
from this document:
 IR = await userManager.AddToRoleAsync(user, role);
if youre still confused in visual studio click on userManager and view definition and you should be able to read exactly how the method works in the comments. obv they don't show you the code because then they would be making themselves vulnerable to hackers

**Comment by Rick-Anderson** [comment]
@Lokesh-Eisen all that code is available on GitHub.

**Comment by Lokesh-Eisen** [comment]
i stand corrected.

**Comment by BenGelton** [comment]
Hi Rick,

I have downloaded the "final2.1" sample from GitHub. 
AspNetCore.Docs/aspnetcore/security/authorization/secure-data/samples/final2.1

It works and I have learned a lot. However it appears that some things are different in .NET Core 2.1 compared to 2.2

I am trying to get my head around how to build a Razor CRUD around the identify framework. Its seams that this would make the whole management of: Users / Roles and User-Role assignment much easier than building your own interface. 

My questions is: Have you got a "final 2.2" version sample written for .NET Core 2.2? that includes an example of this sort of thing? Also is this something that scaffolding could achieve?

Thanks
Ben
